src/ml_gym/frontend/dashboard/src/features/experimentConfig/experimentConfigSlice.tsx

```diff
@@ -0,0 +1,42 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { ExperimentConfigMessageCollectionType, ExperimentConfigType} from '../../app/datatypes'
+import type { RootState } from '../../app/store';
```

Comment:
Codacy has a fix for the issue: [Replace `'../../app/store'` with `"../../app/store"`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=10020887)
```suggestion
import type { RootState } from "../../app/store";
```